zerver/views/muting.py

```diff
@@ -44,7 +45,11 @@ def mute_topic(
 
 
 def unmute_topic(
-    user_profile: UserProfile, stream_id: Optional[int], stream_name: Optional[str], topic_name: str
+    request: HttpRequest,
+    user_profile: UserProfile,
+    stream_id: Optional[int],
+    stream_name: Optional[str],
+    topic_name: str,
```

Comment:
I think for these view helpers, it'd be better to just change the interface for this function to just return None (and have the caller do the `json_success`; it only fails via throwing an exception anyway)